src/main/application/BorosTcf.js

```diff
@@ -1,6 +1,6 @@
 import {inject} from '../core/ioc/ioc'
 
-import {GetVendorListUseCase} from './services/vendor_list/GetVendorListUseCase'
+import {GetVendorListUseCase} from './services/vendorlist/GetVendorListUseCase'
 import {GetConsentStatusUseCase} from './services/vendor_consent/GetConsentStatusUseCase'
```

Comment:
In BorosCmp the services were called 
vendor_list and vendor_consent. 
https://github.com/scm-spain/boros-CMP/tree/master/src/cmp/application/services.

Reply:
I see (didn't look at the old repo at all), but applying here what we do at other projects, can be discussed :)